pipelines.yml

```diff
@@ -127,14 +127,28 @@ pipelines:
       - name: Docker
         type: Bash
         configuration:
+          runtime:
+            type: host
           <<: *COMMON_CONFIGURATION
         execution:
           <<: *UPDATE_COMMIT_STATUS
           onExecute:
             - cd $res_jfrogCliGit_resourcePath
             - docker version
+            # Install Golang
+            - apt update
+            - apt install -yq zip unzip curl git uuid jq gettext golang-1.14-go
```

Comment:
Are you sure you need to install all of these packages? I believe some of them are already installed.